module/VuFind/src/VuFind/Export.php

```diff
@@ -385,4 +386,45 @@ public function getActiveFormats($context = 'record')
         }
         return $this->activeFormats[$context];
     }
+
+    /**
+     * Get the bulk export POST field name.
+     *
+     * @param string $format Format identifier
+     *
+     * @return string
+     */
+    public function getBulkExportPostField($format)
+    {
+        return !empty($this->exportConfig->$format->postField)
+            ? $this->exportConfig->$format->postField : 'ImportData';
+    }
+
+    /**
+     * Get the bulk export POST URL.
+     *
+     * @param string $format Format identifier
+     *
+     * @return string
+     */
+    public function getBulkExportPostUrl($format)
+    {
+        return !empty($this->exportConfig->$format->postUrl)
+            ? $this->exportConfig->$format->postUrl
+            : '';
+    }
+
+    /**
+     * Get the bulk export target window.
+     *
+     * @param string $format Format identifier
+     *
+     * @return string
+     */
+    public function getBulkExportTargetWindow($format)
+    {
+        return !empty($this->exportConfig->$format->targetWindow)
```

Comment:
Is this a new undocumented feature? Should this be noted in export.ini somewhere, or am I just missing something?

Reply:
Yes, comment added.